pkgs/applications/misc/polychromatic/default.nix

```
@@ -1,33 +1,34 @@
-{ lib
-, fetchFromGitHub
-, bash
-, glib
-, gdk-pixbuf
-, gettext
-, imagemagick
-, ninja
-, meson
-, sassc
-, python3Packages
-, gobject-introspection
-, wrapGAppsHook3
-, libappindicator-gtk3
-, libxcb
-, qt5
-, ibus
-, usbutils
+{
+  lib,
+  fetchFromGitHub,
+  bash,
+  glib,
+  gdk-pixbuf,
+  gettext,
+  imagemagick,
+  ninja,
+  meson,
+  sassc,
+  python3Packages,
+  gobject-introspection,
+  wrapGAppsHook,
```

```suggestion
  wrapGAppsHook3,
```

> error: evaluation aborted with the following error message: 'lib.customisation.callPackageWith: Function called without required argument "wrapGAppsHook" at /home/sebtm/.cache/nixpkgs-review/pr-306509/nixpkgs/pkgs/applications/misc/polychromatic/default.nix:14, did you mean "wrapGAppsHook3", "wrapGAppsHook4" or "wrapQtAppsHook"?'

Checked `libayatana-appindicator` still using gtk3 (or 2) ✌🏻

---

Thank you for having a look. I wish I remembered why I made that change. It builds fine for me with either option, but that error message is pretty clear.

Thanks.

---

I suspect your branch is not up2date with master but when running nixpkgs-review and offborg-eval it's rebased I think :)

---

Possible, but I rebased before the new 9.1 update. No reason fort the change anyway, so thank you for noticing that.